modules/ocf_mesos/files/master/dns/mesos-dns.service

```diff
@@ -1,12 +0,0 @@
-[Unit]
-Description=Mesos DNS
```

Comment:
You should also be able to remove this section from our DNS config, since we don't need to forward on DNS queries to this service any more: https://sourcegraph.ocf.berkeley.edu/github.com/ocf/dns@2dae37acd4ae27ec790417b2c0b81fc220587e10/-/blob/etc/named.conf.local#L9-12

Reply:
I plan to go through all the other repos stripping out code later today. Thanks for the information though!